tests/behat/multilang_mastertemplate.feature

```diff
@@ -5,7 +5,8 @@ Feature: verify multilang in ATTLS (20 item version) mastertemplate
   I display a mastertemplate                                                  // The feature we want
 
   Background:
-    Given the following "courses" exist:
+    Given remote langimport tests are enabled
```

Comment:
I've also added this step, so the test will be skipped by default. You need to define:

````php
define('TOOL_LANGIMPORT_REMOTE_TESTS', true);
````

in your config.php to get those tests executed. That's the way
as far as they are using external resources.